whats-new-panel.yaml

```diff
@@ -260,3 +98,63 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_PIP_76
+  template: whatsnew_panel_message
+  order: 3
+  content:
+    bucket_id: WHATS_NEW_PIP_76
+    published_date: 1586859903200
```

Comment:
Is this a new PIP message? If not, shall we stick with the `published_date` of the current PIP?

Reply:
New message, it will replace the old one but we can't switch until the release day.

Reply:
Or actually I can add version targeting to the old one.